daemon/algod/api/client/restClient.go

```diff
@@ -64,6 +64,19 @@ var rawRequestPaths = map[string]bool{
 	"/v2/teal/compile": true,
 }
 
+// unauthorizedRequestError is generated when we receive 401 error from the server. This error includes the inner error
+// as well as the likely parameters that caused the issue.
+type unauthorizedRequestError struct {
+	errorString string
+	apiToken    string
+	url         string
+}
+
+// Error format an error string for the unauthorizedRequestError error.
+func (e unauthorizedRequestError) Error() string {
+	return fmt.Sprintf("Unauthorized request to `%s` when using token `%s` : %s", e.url, e.apiToken, e.errorString)
```

Comment:
Why not embed filterASCII in this function?

Reply:
I wanted to use the filtered string for the other errors as well. We shouldnt have errors like that with unfiltered content since its being written to the console.